test/integration/dns_test.go

```diff
@@ -21,7 +22,16 @@ func TestDNS(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 
-	localIP := net.ParseIP("127.0.0.1")
+	localAddr := ""
```

Comment:
@pweil- I'm concerned about the fact that it changed.  I didn't see an obvious reason why.

@smarterclayton You weren't expecting something to change in the dns server, were you?